compiler/rustc_trait_selection/src/traits/const_evaluatable.rs

```diff
@@ -567,19 +567,19 @@ pub(super) fn try_unify_abstract_consts<'tcx>(
     // on `ErrorReported`.
 }
 
-pub fn walk_abstract_const<'tcx, F>(
+pub fn walk_abstract_const<'tcx, F, R>(
```

Comment:
```suggestion
pub fn walk_abstract_const<'tcx, R, F>(
```
nit: I prefer putting unnameable types last